src/hit/api/evaluator/homomorphic.cpp

```diff
@@ -127,6 +116,19 @@ namespace hit {
         delete seal_decryptor;
     }
 
+    void HomomorphicEval::makeSealCtxt(const seal::EncryptionParameters &params, const timepoint &start) {
+        if (standard_params_) {
+            context = make_unique<SEALContext>(params);
+            print_elapsed_time(start, "Creating encryption context...");
+        } else {
+            LOG(WARNING) << "YOU ARE NOT USING STANDARD SEAL PARAMETERS. Encryption parameters may not achieve 128-bit security"
+                         << "DO NOT USE IN PRODUCTION";
+            // for large parameter sets, see https://github.com/microsoft/SEAL/issues/84
+            context = make_unique<SEALContext>(params, true, sec_level_type::none);
+            print_elapsed_time(start, "Creating encryption context...");
+        }
+    }
```

Comment:
Nit: duplicate of `print_elapsed_time(start, "Creating encryption context...");`. It can be put in last line of this method.